benchmarks/htap/schemas/s64da_native_enhanced/s64da_indexes.sql

```diff
@@ -45,34 +45,3 @@ CREATE INDEX idx_order_line_cache ON order_line USING columnstore (
   , ol_amount
   , ol_dist_info
 );
-
```

Comment:
why is that?

Reply:
the removal of these indexes you mean? its because they are on tables that are e.g. 1MB, then a columnstore brings no value.